instances/instance_delete.go

```diff
@@ -16,6 +16,7 @@ import (
 
 // Deletes the specified instance. After this call the UUID of the instance is
 // no longer valid. If the instance is currently running it is force stopped.
+// UUIDs can also be names.
 //
 // See: https://docs.kraft.cloud/002-rest-api-v1-instances.html#delete
 func (c *instancesClient) Delete(ctx context.Context, uuid string) error {
```

Comment:
Let's update the method's argument `uuid` to `uuidOrName`, such that it's visible by the LSP